www/data_module/src/classes/base.service.spec.js

```diff
@@ -5,7 +5,7 @@
  */
 describe('Base class', function() {
     let $q, dataService, socketService;
-    beforeEach(module('bbData'));
+    beforeEach(angular.module('bbData'));
```

Comment:
I think module in the spec is different from angular.module:
https://docs.angularjs.org/api/ngMock/function/angular.mock.module